Casks/m/marshallofsound-google-play-music-player.rb

```diff
@@ -7,6 +7,8 @@
   name "Google Play Music Desktop Player"
   homepage "https://www.googleplaymusicdesktopplayer.com/"
 
+  deprecate! date: "2024-07-27", because: :unmaintained
```

Comment:
```suggestion
  disable! date: "2024-07-27", because: :no_longer_available
```

Google Play Music has been shut down ([source](https://music.youtube.com/googleplaymusic)), so we might as well disable the cask.